api/_key.js

```diff
@@ -2,36 +2,36 @@ import { strict as assert } from 'assert';
 import { readFileSync } from 'fs';
 
 const loadFromEnv = (varName) => {
-	const value = process.env[varName];
-	if (typeof value !== 'string') {
-		throw new Error(`Environment var "${varName}" is unset`);
-	}
-	const data = Buffer.from(value, 'base64').toString('utf8');
+    const value = process.env[varName];
```

Comment:
Please remove changes to this file

Reply:
Should be just changes from the eslint branch now